WcaOnRails/db/migrate/20180120132926_add_board_team.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddBoardTeam < ActiveRecord::Migration[5.1]
+  def change
+    execute <<-SQL
+      INSERT INTO teams (friendly_id, email, rank, created_at, updated_at) values ('board', 'board@worldcubeassociation.org', 1, NOW(), NOW());
```

Comment:
Don't we need this migration (or a new migration) to change all `User.delegate_status` from `board_member` to `delegate`, and populate the team members table with that information? Similarly code like:

- https://github.com/jfly/worldcubeassociation.org/blob/05670a20bf5e6e659a7bf42a6f4d6a56cf7b33f8/WcaOnRails/app/models/user.rb#L70
- https://github.com/jfly/worldcubeassociation.org/blob/05670a20bf5e6e659a7bf42a6f4d6a56cf7b33f8/WcaOnRails/app/models/user.rb#L306

can get deleted.